CHANGELOG.md

```diff
@@ -74,6 +68,7 @@ exception). It is recommended to run this tool for existing projects (PR #5270)
 
 ### Notes For Existing Projects
 - PHP should be upgraded to 7.3 to before upgrading LORIS.
+- For dev instances, php7.3-curl is now a required dependency.
```

Comment:
@AlexandraLivadas why only for DEV ?

Reply:
I think you mean Laetitia @laemtl 

@johnsaigle's PR header in #6187 suggests this is only an issue on Dev environments. 
John was also [asked for clarification on this exact point](https://github.com/aces/Loris/issues/6473#issuecomment-624948602) but not response so far... 

@ridz1208 Are you confirming that it applies to production too or just asking?

Reply:
from what I can see it is indeed just a dev thing. I think it's still worth properly testing however

Reply:
👍 thanks @ridz1208 this might need shen's help with the install VMs

Reply:
@ridz1208 @christinerogers This is correct, php-webdriver requires the php7.3-curl in the dev environment.

Reply:
@ridz1208 It's definitely only a dev requirement. https://github.com/aces/Loris/pull/6187 updates a package that was renamed. This package requires php7.X-curl. That package drives our integration tests. If you're not running integration tests, the package and its dependency are not required. It's only installed when someone runs `make dev`.

Reply:
thanks for confirming @johnsaigle